src/nvim/os/env.c

```diff
@@ -76,6 +76,31 @@ char *os_getenvname_at_index(size_t index)
   return name;
 }
 
+/// Check if the environment variable `name` exists and has
+/// a value of either 1, "on", "yes", or "true".
+bool os_env_istruthy(const char *name) FUNC_ATTR_NONNULL_ALL
+{
+  const char *val = os_getenv(name);
+  if (val == NULL || *val == NUL) {  // TODO(Pyrohh): Remove the check for
+    return false;                    // NUL after #2574 is merged.
+  }
+
+  errno = 0;
+  long n = strtol(val, NULL, 10);
```

Comment:
> Isn't that already being done below? And yes, as far as I know it is.

I meant replace the strtol with just a string comparison. Although in theory that might end failling due to some encoding.